### PR TITLE
Add test coverage for BlockingStorage

### PR DESCRIPTION
**Pull request by dgarus**
Part of #55 
 - add test coverage for BlockingStorage

### CODE REVIEW

File: src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        final Key destination = new Key.From("shouldMove-destination");
+        this.original.save(source, new Content.From(data)).join();
+        this.blocking.move(source, destination);
+        MatcherAssert.assertThat(
+            new Remaining(
+                this.original.value(destination).thenApplyAsync(
+                    pub -> new Concatenation(pub).single().blockingGet()
+                ).join(),
+                true
+            ).bytes(),
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java:96]: `PublisherAs` can be used here for simplification

File: src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            Matchers.equalTo(data)
+        );
+    }
+
+    @Test
+    void shouldDeleteInOriginalWhenKeyIsDeletedByBlocking() throws Exception {
+        final Key key = new Key.From("test_key_6");
+        this.original.save(key, new Content.From("some data6".getBytes())).join();
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java:104]: as content value does not matter here, I'd suggest using `Content.EMPTY` to simplify

File: src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        MatcherAssert.assertThat(
+            keys,
+            Matchers.equalTo(Arrays.asList("a/b/2", "a/b/c/1"))
+        );
+    }
+
+    @Test
+    void shouldExistInOriginalWhenKeyIsSavedByBlocking() throws Exception {
+        final Key key = new Key.From("test_key_2");
+        final byte[] data = "test data2".getBytes();
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/asto/blocking/BlockingStorageTest.java:75]: `data` can inlined or `Content.EMPTY` can be used here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, check several comments please

**Comment by dgarus** [comment]
@olenagerasimova, thank you for your review.
I fixed the comments, have please a look.

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks

**Pull request merged**